extensions/Worker.Extensions.Abstractions/src/RetryAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    public abstract class RetryAttribute : Attribute
+    {
+        public RetryAttribute()
```

Comment:
Do we need this public constructor?

Reply:
Intent was to stylistically match the other attribute classes ([binding attribute](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.Abstractions/src/BindingAttribute.cs), [input binding attribute](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.Abstractions/src/InputBindingAttribute.cs), etc).